src/main/java/uk/gov/hmcts/probate/service/CaseQueryService.java

```diff
@@ -89,8 +89,14 @@ private static <T> T nonNull(@Nullable T result) {
 
     public List<ReturnedCaseDetails> findGrantIssuedCasesWithGrantIssuedDate(String invokedFrom, String queryDate) {
         BoolQueryBuilder query = boolQuery();
+        BoolQueryBuilder stateChecks = boolQuery();
 
-        query.must(matchQuery(STATE, STATE_MATCH));
+        for (String stateToMatch : Arrays.asList(STATE_MATCH)) {
```

Comment:
I think there's term query would be faster that match (cos it will be looking at variants not just exact) but maybe we aren't currently using term query?
eg QueryBuilders.termQuery as opposed to QueryBuilders.matchQuery (which does the same as your new MatchQueryBuilder)

Reply:
Probably something for the back burner - I only see these match queries currently, they might well be mapped as text which needs match :)